### ISSUE TITLE
Notifications on internal links

### ISSUE THREAD

**Issue by abarbee** [kanboard/kanboard#2755]
currently it is not possible to receive notifications when a task is blocked by another task.

in Settings > Task's links
it would be interesting when we add a link to enable notifications.

when a Task 1 is related to Task 2, it would be interesting to be notified changes.
for example 
"task 1 is blocked by task 2"
when the Task 2 is closed, the person assigned to the task 1 should be notified.

that would be great

**Comment by bugwatcher** [comment]
This issue has been closed automatically for inactivity after more than 1 year. If you still need this feature, you should consider contributing to this open-source project.
